lib/active_storage_validations/processable_image_validator.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
 require_relative 'concerns/active_storageable.rb'
+require_relative 'concerns/attachable.rb'
 require_relative 'concerns/errorable.rb'
-require_relative 'concerns/metadatable.rb'
 require_relative 'concerns/symbolizable.rb'
 
 module ActiveStorageValidations
   class ProcessableImageValidator < ActiveModel::EachValidator # :nodoc
     include ActiveStorageable
+    include Attachable

```

Comment:
<img width="488" alt="image" src="https://github.com/user-attachments/assets/e9d01d90-b2ba-46fe-8f7a-edaddb9a7fef">

I have a concern in my application located in `app/models/concerns/attachable`, and since Zeitwerk has a special case for the `concerns/*.rb` directory when autoloading/eager loading, 

==> it picks up this gem's before mine

`lib/active_storage_validations/concerns/attachable.rb` ( probably because `active_storage_validations` comes before `app` alphabetically )

Do you think it's possible to rename the directory `concerns` to something else please? @igorkasyanchuk

Reply:
As a workaround, I need to do this for now `include ::Attachable` in my models :/

Reply:
Hi @chaadow,
I did not think `Zeitwerk` would pick up the gem concerns... That's something I was unaware of, I am going to fix this issue. You can still downgrade to v1.3.0 while waiting the fix. :)

Reply:
Issue https://github.com/igorkasyanchuk/active_storage_validations/issues/281 opened to solve this